include/seqan3/core/type_list/traits.hpp

```diff
@@ -132,6 +145,27 @@ inline constexpr ptrdiff_t count =  (SEQAN3_IS_SAME(query_t, pack_t) + ... + 0);
 template <typename query_t, typename ...pack_t>
 inline constexpr ptrdiff_t find = seqan3::pack_traits::detail::find<query_t, pack_t...>();
 
+/*!\brief Get the index of the first type in a pack that satisfies the given predicate.
+ * \tparam pred_t   The type you are searching for.
+ * \tparam pack_t   The type pack.
+ * \returns The position or `-1` if no types match.
```

Comment:
```suggestion
 * \returns The index or `-1` if no types match.
```
You call it index in the brief. change both to `position` otherwise